playroom/components.tsx

```diff
@@ -319,7 +321,7 @@ export const PreviewTools: React.FC<PreviewToolsProps> = ({
         </ThemeContextProvider>
     );
 
-    if (floating) {
+    if (floating && !hide) {
```

Comment:
instead of adding the `&& hide` here, I'd add an early return before this

Reply:
perfect!